recipes/exiv2/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class Exiv2Conan(ConanFile):
+    name = "exiv2"
+    description = "Exiv2 is a C++ library and a command-line utility " \
+                  "to read, write, delete and modify Exif, IPTC, XMP and ICC image metadata."
+    license = "GPL-2.0"
+    topics = ("conan", "image", "exif", "xmp")
+    homepage = "https://www.exiv2.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_png": [True, False],
+        "with_xmp": [True, False],
+        "with_curl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_png": True,
+        "with_xmp": True,
+        "with_curl": False,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("libiconv/1.16")
+        if self.options.with_png:
```

Comment:
https://github.com/Exiv2/exiv2/blob/v0.27.4/CMakeLists.txt#L18
It says that `with_png` needs `libz`. 
Should we add it here or is that a transitive dependency of `libpng`?

Reply:
Indeed, our libpng recipe needs zlib:
https://github.com/conan-io/conan-center-index/blob/f6faf7c4461c386fc40cf52a77f0b65357c66d9c/recipes/libpng/all/conanfile.py#L79

Reply:
yes, zlib is transitive dep of libpng, no need to add explicitly. PNG files are more or less lossless images compressed with zlib.